packages/business-rules/src/constants.js

```diff
@@ -84,7 +84,8 @@ const NEW_OR_SAVED_APPEAL_OPTION = {
 const DECISION_OUTCOME = {
```

Comment:
can we pull things like this from enums in data model instead having a copy

Reply:
I am an advocate of adding as much shared stuff as possible to the data model repo, or some other shared repo so yes

Reply:
Yes maybe we get generated JS constants from data-model repo + some generated OpenAPI spec objects.

Reply:
ugh I'm just doing it now